### DISCUSSION TITLE
[Shop] Add Subcategories/Categories

### DISCUSSION THREAD

**Discussion by ScaredDonut** [gameserverapp/Platform#646] [Suggestions]
Hi,

I think it'd be good if you can add categories/subcategories to the shop system on the admin side and client side, furthermore, the ability to add folders/categories so you can manage packs by specific game.

You should also be able to sort the categories by labels

**Comment by mvaessen** [comment]
Hi,

Thanks for your suggestions. Please create separate suggestion posts for each suggestion so people can vote per suggestion.

**Reply by ScaredDonut** [reply]
Sure! consider it done :)
